src/jit/morph.cpp

```diff
@@ -16687,6 +16687,8 @@ void Compiler::fgMorph()
 
     fgUpdateFinallyTargetFlags();
 
+    fgTailMergeThrows();
```

Comment:
Seems like you should make this a real PHASE so its cost gets counted separately.

Reply:
Done.